src/helm/benchmark/presentation/run_specs_opinions_qa_openai_default.conf

```diff
@@ -0,0 +1,98 @@
+entries: [
+{description: "opinions_qa:survey_type=Pew_American_Trends_Panel_disagreement_500,context=default,num_logprobs=100,model=openai/ada,num_train_trials=1", priority: 1}
```

Comment:
Can use RunExpander to reduce the number of lines here too to one for all the models.

Reply:
👍